docs/reference/cli/cli.md

```diff
@@ -0,0 +1,20 @@
+---
+title: CLI Tools
```

Comment:
Please move the whole CLI TOOLS folder up a level. Hiding it under Reference is not helpful. I specifically removed double deep folders when I reworked the docs website structure.

Reply:
Hmm. If we're going to do that, then I want to move the server and agent docs under this same section, as they are technically subcommands the same as everything else that we've got in this section.

I also moved the HA stuff under "Cluster Datastore Options", maybe we should move that up to the top-level and put the backup/restore stuff in with it.

Reply:
+1 on moving the Cluster Datastore up to the top and adding the backup sections to it. 

I can add to the README in my next docs PR around convention.